src/app/detail/ui/TaskEditor.tsx

```diff
@@ -173,6 +174,7 @@ export const TaskEditor = ({
             return t
           }}
           deleteEditorAttachments={(url) => deleteEditorAttachmentsHandler(url, token ?? '', task_id)}
+          attachmentLayout={(props) => <AttachmentLayout {...props} />}
```

Comment:
any reason we spread the AttachmentLayout and dont directly do `attachmentLayout={AttachmentLayout}`?

Reply:
This is done because we want the props to be dynamically passed from parent component(Tapwrite) to child component (Task App's task editor). We are to get props from Tapwrite like fileName, src, fileSize, fileType because file handling is done through Tapwrite.

Reply:
What do you mean by dynamically passed?

If you do `attachmentLayout={AttachmentLayout}` the props are passed down are they not?

Reply:
yup. it should work, sorry I misunderstood. Applying the changes.